code/src/layouts/header/components/header-toolbar/HeaderToolbar.test.tsx

```diff
@@ -3,13 +3,17 @@ import { fireEvent, screen } from "@testing-library/react";
 import HeaderToolbar from "@/layouts/header/components/header-toolbar/HeaderToolbar";
 
 import { useAppDispatch } from "@/hooks/use-redux/useRedux";
-import { logout, useIsAuthSelector } from "@/store/slices/userSlice";
+import {
+  logout,
+  useIsAuthLoadingSelector,
+  useIsAuthSelector
+} from "@/store/slices/userSlice";
 import renderWithProviders from "@/utils/render-with-providers/renderWithProviders";
 
 jest.mock("@/store/slices/userSlice", () => ({
   __esModule: true,
-  default: () => ({}),
```

Comment:
I guess this line was added to remove warning during tests run. Consider leaving it as it was, because warning appears again and it makes test result messy

![image](https://github.com/idx-academy/spa-orders/assets/62070431/f6de8303-d9c2-4d6e-9f25-ad942d1263a9)

Reply:
> I guess this line was added to remove warning during tests run. Consider leaving it as it was, because warning appears again and it makes test result messy
> 
> ![image](https://private-user-images.githubusercontent.com/62070431/346640999-f6de8303-d9c2-4d6e-9f25-ad942d1263a9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MjA0NTcyNzMsIm5iZiI6MTcyMDQ1Njk3MywicGF0aCI6Ii82MjA3MDQzMS8zNDY2NDA5OTktZjZkZTgzMDMtZDljMi00ZDZlLTlmMjUtYWQ5NDJkMTI2M2E5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNDA3MDglMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjQwNzA4VDE2NDI1M1omWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTc4NWVjYzVmNTlkOGI2MWU1NjcyYzFjYTgzN2IyNWRmMzUxMTA0NDA4MzBhNzhjZWVlODZhYmZjMTE5NWI5MzAmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0JmFjdG9yX2lkPTAma2V5X2lkPTAmcmVwb19pZD0wIn0.GqU1Squ0jVkCIx75SyVPH5XY8dBlwuOmyXuSf7miSlc)

Returned it. Thanks